JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -92,14 +93,15 @@ module Microsoft.ApplicationInsights {
                 this.session = new Context.Session();
                 this.sample = new Context.Sample(config.sampleRate());
             }
+
+            this._addDefaultTelemetryInitializers();
```

Comment:
Do we need to provide a way to disable this initializer for someone who wants to get browser link telemetry or someone who has similarly routes in the app?